analyses/survival-analysis/02-summary-statics-TP53.Rmd

```diff
@@ -14,6 +14,7 @@ suppressWarnings({
   library(ggplot2)
   library(patchwork)
   library(ggpubr)
+  library(ggalluvial)
```

Comment:
Since this PR hasn't been merged yet, I just installed locally and the script ran successfully.